crates/core/src/lib.rs

```diff
@@ -14,7 +14,7 @@ pub fn stdb_path<S>(s: &S) -> PathBuf
 where
     S: AsRef<Path> + ?Sized,
 {
-    STDB_PATH.join(s)
+    dbg!(STDB_PATH.join(s))
```

Comment:
This looks stray.

Reply:
Ups, I address it at https://github.com/clockworklabs/SpacetimeDB/pull/178